### PR TITLE
Fix doc typos

### PR DESCRIPTION
**Pull request by MuzafferDede**
🍭

### CODE REVIEW

File: packages/docs/src/en/upgrade-guide.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Upgrade From V2
 
 # Upgrade from V2
 
-Upgrading from Alpine V2 to V3 should be fairly painless. In many cases, NOTHING has to be done to your codebase to use V3. Below is an exhaustive list of breaking changes and deprecations in descending order of how likely users are to be affected by them:
+Upgrading from Alpine V2 to V3 should be fairly painless. In many cases, NOTHING has to be done to your codebase to use V3. Below is an exhaustive list of breaking changes and depreciations in descending order of how likely users are to be affected by them:
```
Comment (adevade) [inline packages/docs/src/en/upgrade-guide.md:8]: This should not change.

[Depreciation](https://en.wikipedia.org/wiki/Depreciation)
> This article is about the concept in accounting and finance involving fixed capital goods.

 [Deprecation](https://en.wikipedia.org/wiki/Deprecation)
> While a deprecated software feature remains in the software, its use may raise warning messages recommending alternative practices.

**Review by adevade — CHANGES_REQUESTED** [review]
**Deprecations** should not be changed.

File: packages/docs/src/en/upgrade-guide.md (hunk 1 above)
Comment (MuzafferDede) [reply packages/docs/src/en/upgrade-guide.md:8]: Yeah, just noticed it. Thanks

**Comment by calebporzio** [comment]
Thanks! (I added this fix manually because this repo has been moved - sorry about that!)

**Pull request closed without merging**